app.py

```diff
@@ -5,24 +5,51 @@
 
 
 def get_answer(question, answer, options):
+    '''File given answer'''
+
     global questions
     global SEPARATOR
     answer = options.split(SEPARATOR)[answer]
-    answers.update({question:int(answer == questions.Answer[questions.Question == question].iat[0])})
+    questions['Choice'][questions.Question == question] = answer
+
+
+def find_score(column_name):
+    '''Count chosen answers in a given column'''
+
+    global questions
```

Comment:
Might be preferable to pass variables as arguments for functions that we want to test